### PR TITLE
Data model

### PR DESCRIPTION
**Pull request by Kaiserabbas**
# Create a new app "blog"
- Initialized project with Git.
- Made sure that your project has Postgres database set up.
- Build project schema.
- Created and run the necessary migration files.
- Table and column names match the ERD diagram.
- Posts table the column holding the foreign key to the users table named author_id
- All columns that are foreign keys have a corresponding index.

### CODE REVIEW

File: db/migrate/20231108095514_create_posts.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  def change
+    create_table :posts do |t|
+      t.references :author, null: false, foreign_key: true
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+
+      t.timestamps
+    end
```
Comment (AbelG101) [inline db/migrate/20231108095514_create_posts.rb]: - [ ] Great work so far! Just a quick note: when creating a foreign key with a column name that differs from the table name, like in line `4`, Rails may not know which table to refer to by default. To resolve this, it's important to explicitly specify the table name, as demonstrated in the code snippet below:
```rb
 t.references :author, foreign_key: { to_table: 'users' }
```

File: README.md [hunk 2]
```diff
@@ -1,24 +1,112 @@
...
+
+I would like to thank Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](/LICENSE) license.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (AbelG101) [inline README.md]: - [ ] I also noticed your MIT link is broken so kindly make sure to fix this as it makes your repo look unprofessional 👶

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @Kaiserabbas 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Migrations are properly created ✔
- Most associations are properly implemented ✔
- Correct use of Git Flow ✔
- All linter checks are passing ✔
- Descriptive PR title with a good summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20231108095514_create_posts.rb (hunk 1 above)
Comment (Kaiserabbas) [reply db/migrate/20231108095514_create_posts.rb]: Thank you, This has been updated accordingly

File: README.md (hunk 2 above)
Comment (Kaiserabbas) [reply README.md]: Licence file is created and link has been updated. Thank You

**Review by Olamarx — APPROVED** [review]
Hi @Kaiserabbas, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️
- Changes requested fixed ✔️
- Commit messages meaningful ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**